engine/src/main/java/org/destinationsol/game/Hero.java

```diff
@@ -40,11 +41,11 @@
     private boolean isTranscendent;
     private boolean isDead;
 
-    public Hero(SolShip shipHero) {
+    public Hero(SolShip shipHero, SolGame solGame) {
```

Comment:
Watch out - this is a potentially _big_ change. You need to be careful about what code you might affect when you do things like add or subtract arguments on heavily used objects. It does seem like you found the other spots where `Hero` is constructed, but there are suddenly 27 unit tests that fail, indicating something went really wrong. This might relate

See http://jenkins.terasology.org/view/DestSol/job/DestinationSolPRs/697/testReport/